draft-irtf-cfrg-opaque.md

```diff
@@ -1070,12 +1236,28 @@ such that the following conditions are met:
 - The MHF has fixed parameters, chosen by the application, and implements the
   interface in {{dependencies}}. Examples include Argon2 {{?I-D.irtf-cfrg-argon2}},
   scrypt {{?RFC7914}}, and PBKDF2 {{?RFC2898}} with fixed parameter choices.
-- EnvelopeMode value is as defined in {{credential-storage}}, and is one of
+- EnvelopeMode value is as defined in {{client-credential-storage}}, and is one of
   `base` or `custom_identifier`.
 - The Group mode identifies the group used in the OPAQUE-3DH AKE. This SHOULD
   match that of the OPRF. For example, if the OPRF is OPRF(ristretto255, SHA-512),
   then Group SHOULD be ristretto255.
 
+To recover a public key from a private key, OPAQUE-3DH implements `RecoverPublicKey(private_key)` as follows:
```

Comment:
I'm not sure how I feel about this function here. It seems sort of out of place. But we can keep it now and move it around later.

Reply:
Should we move this to the _dependencies_ section?